### ISSUE TITLE
Problème automatisation code douane

### ISSUE THREAD

**Issue by lgiRomain** [lgi-wines/Extranet-LGI#183]
### Description
Sur la fiche produit 31177, le code douane sur l'onglet "Formulaire" ne s'automatise pas

### Scénario pour reproduire l'anomalie
Sur cette fiche, j'ai modifiés les composants, et sur les composants "Vin" et "Contre étiquette", je suis passé de l'Appellation "IGP Pays d'Oc" à "VDF Chardonnay"

### Résultat attendu
Lors de l'automatisation, le code douane devrait s'automatiser pour devenir 2204 21 81 1 (cf CustomCodeID=683)

### Anomalie constatée
Le code douane reste celui de l'IGP Pays d'Oc blanc 2204 21 79 3

**Comment by Portekoi** [comment]
Effectivement, le code douane n'est calculé que lors d'un clic sur le bouton `Automatiser` qui n'apparait que quand le produit n'est pas actif. 

Il semble ne pas avoir été prévu de pouvoir modifier ce code une fois le produit activé.

**Comment by lgiRomain** [comment]
Le problème d'automatisation se produit quand je passe la fiche en statut "Projet" et que je clique sur le bouton "Automatiser". Le code douane ne se change pas

**Comment by Portekoi** [comment]
OK. Je pensais pas pouvoir mettre ce produit en "Projet". Je le prends en débogue et je te dis.
